commands-yml/commands/interactions/touch/double-tap.yml

```diff
@@ -0,0 +1,80 @@
+---
+name: Double Tap
+short_description: Double tap on the touch screen using finger motion events
+   
+example_usage:
+  java:
+    |
+      TouchActions action = new TouchActions(driver);
+      action.doubleTap(element);
+      action.perform();
+  python:
+    |
+      actions = TouchActions(driver)
+      actions.double_tap(element)
+      actions.perform()
+  javascript_wd:
+    |
+      // Using tapElement method
+      await driver.tapElement(elementOne);
+
+      // Using touch actions
+      let action = new wd.TouchAction();
+      action.tap({el: element});
+      action.perform();
+      await driver.performTouchAction(action);
+  ruby:
+    |
+      @driver.touch_action.double_tap(element).perform
+  php:
+    |
+      // TODO PHP sample
+  csharp:
+    |
+      // TODO C# sample
+
+client_docs:
+  java: "https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/interactions/touch/TouchActions.html#doubleTap-org.openqa.selenium.WebElement-"
+  python: "https://seleniumhq.github.io/selenium/docs/api/py/webdriver/selenium.webdriver.common.touch_actions.html#selenium.webdriver.common.touch_actions.TouchActions.double_tap"
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L425"
+  ruby: "http://www.rubydoc.info/gems/selenium-webdriver/Selenium%2FWebDriver%2FTouchActionBuilder:double_tap"
+  php: "https://github.com/appium/php-client/" # TODO PHP documentation link
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO Dotnet documentation link
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: true
+    uiautomation: true
+  android:
+    uiautomator2: true
+    uiautomator: true
```

Comment:
Isn't it false? we use TouchAction API for doubletap not TouchActions.

Reply:
So it's not supported in Android?

Reply:
yes. `doubleTap` is supported via TouchAction.

Reply:
Should it be:

```
 android:
    uiautomator2: true
    uiautomator: true
```

Reply:
Should be:
```
ios:
   xcuitest: false
   uiautomation: false
android:
  uiautomator2: false
   uiautomator: false
```

Reply:
It should be like above for this whole PR everywhere